index.html

```diff
@@ -138,17 +139,11 @@ <h2>
                 <button class="btn contact-btn" type="submit" name="btn">Get in touch</button>
```

Comment:
- [ ] Please the error message should show up close to the "Get in touch" button.
-
![good](https://user-images.githubusercontent.com/63972670/168339569-e799d0f7-4c43-47c5-8c2f-372691e4d6ac.png)

Reply:
hello, thanks for your input. But there is a title for this PR "Contact Form Validation Review #14"

Reply:
> hello, thanks for your input. But there is a title for this PR "Contact Form Validation Review #14"
![s](https://user-images.githubusercontent.com/63972670/168376153-aef4ba72-44ed-4836-a35f-a0af431e95a8.PNG)
- Please there is a link connected with "in a professional way" That is why it is in blue color. Kindly click here to learn more ([in a professional way](https://github.com/microverseinc/curriculum-transversal-skills/blob/main/documentation/articles/professional_repo_rules.md))